_includes/specials/taproot/en/18-trivia.md

```diff
@@ -0,0 +1,83 @@
+  part to circumvent the [patent on schnorr signatures][schnorr patent]
+  and it was [reported][gov report] that "there were allegations of
+  infringement from Mr. Claus Schnorr and Public Key Partners (PKP)"
+  that DSA violated Schnorr's patent.  That patent expired in 2008.
+
+- **Un-Schnorr what name to use:** although it didn't work out, there
+  was a [suggestion][dryja bn sigs] early in the development of adapting
+  schnorr signatures for Bitcoin that Claus Schnorr's name shouldn't be
+  used in association with them because his patent on them prevented the
+  widespread use of a valuable cryptographic technique for over 20
+  years.
+
+- **Schnorr for twisted Edwards curves:** an application of schnorr
+  signatures using elliptic curves was written in 2011 <!-- guessing
+  from URL; Wikipedia also says this without a source --> and
+  [published][ed25519] in 2012<!-- verified -->.  The scheme,
+  [EdDSA][], is now the basis of several standards.  Although not used
+  in Bitcoin consensus, references to it in the context of other systems
+  can be found in many of the Bitcoin repositories tracked by Optech.
+  <!-- source: quick git grep -i -->
+
+- **Pay to contract:** Ilja Gerhardt and Timo Hanke created a
+  [protocol][gh p2c], presented by Hanke at the 2013 San Jose Bitcoin
+  conference, <!-- source: Wuille; I found some independent confirmation in dead links on Google -harding -->
+  for allowing a payment to commit to the hash of its contract.  Anyone
+  with a copy of the contract, and the nonce used to avoid certain
+  attacks, can verify the commitment---but to anyone else the payment
+  looks like any other Bitcoin payment.
+
+    A slight improvement to this *pay-to-contract* (P2C) protocol was
+    included in the [2014 paper][sidechains.pdf] about
+    [sidechains][topic sidechains], <!-- Algorithm 1:
+    GenerateCrossChainAddress --> where the commitment also includes the
+    original public key to pay.  Taproot uses this same construction
+    but, instead of committing to the terms of an offchain contract, the
+    output creator commits to consensus-enforced terms chosen by the
+    receiver for how they can spend the received bitcoins onchain.
+
+- **A Good Morning:** the idea to use P2C so that payments to scripts
+  can look identical onchain to paying public keys was invented in Los
+  Altos, California, at the diner "A Good Morning" on 22 January 2018.
+  Pieter Wuille writes that the idea was developed by Andrew Polestra
+  and Gregory Maxwell "while I briefly left the table... !$%@" [sic].
+  <!-- personal correspondence with Wuille -harding -->
+
+- **<!--x-->2.5 years in 1.5 days:** choosing the optimal constant for
+  [bech32m][topic bech32] required [about][wuille matrix elimination]
+  2.5 years of CPU time, which was performed in just 1.5 days mostly
+  using a CPU cluster belonging to Gregory Maxwell.
+
+*We thank Gregory Maxwell and Pieter Wuille for enjoyable conversations
```

Comment:
Maybe @ajtowns, @jonasnick or @real-or-random have some more trivia about the history of the proposal?